tensorflow-core/tensorflow-core-api/src/main/java/org/tensorflow/types/TFloat64.java

```diff
@@ -29,10 +29,11 @@
 import org.tensorflow.ndarray.NdArray;
 import org.tensorflow.ndarray.StdArrays;
 import org.tensorflow.ndarray.impl.dense.DoubleDenseNdArray;
+import org.tensorflow.types.family.TFloating;
```

Comment:
Provably that the `TNumber` import is not required anymore?

Reply:
@karllessard I don’t fully understand your comment. Did you mean `TNumber` is not required for most Activations or not required at all? `TFloating` works well with those activations that only work with floating point. But there is still use for `TNumber` when any number type is accepted.

Reply:
No, I just meant that in these type classes `TFloat32, TFloat16, etc.`, the `TNumber` seems not to be used anymore since you replaced it by `TFloating` therefore there is no need to import it :)

Reply:
actually my comment should have been put on the line below... sorry, that is kind of things that happens when you review a PR on a iPad

Reply:
Interesting the `google-java-format` did not remove the `TNumber` import, I had to remove them all by hand.